### ISSUE TITLE
2.4.3.3 M2 Report Sequence Diagram

### ISSUE THREAD

**Issue by donhitme34** [INF2001/TEAM-PROJECT-P11-3#100]

**Comment by jasonyjk** [comment]
1. Task title ID: **#100**
2. What is the task about? **The task involves documenting the writeup for sequence diagrams UC-8 Enrol New Students for Class and UC-12 Allocate job assignments.**
3. What are the goals of the task? The goal of this task is to detail how system components interact during student enrollment and job allocation. With the explanation writeup aided by visual guides, it serve as blueprints for understanding workflows, roles, and interactions within the system.
4. Success or completion criteria. Success is achieved when the written explanation for the sequence diagrams is logical and aligns with the use case descriptions and design specifications, while also being clear and comprehensible to team members and stakeholders.
5. Allocated time: **144 hours**
6. Start date and end date of the issue/task **20/10/2023 to 25/10/2023**
7. Who is the owner? **Jason**
8. The status of the tasks. **Completed**
9. What is the Priority of the task? **HIGH**

**Comment by jasonyjk** [comment]
1. In our detailed work process, we have successfully wrote the explanation for sequence diagram UC-8 Enrol New Students for Class and UC-12 Allocate job assignments. Work has also been peer-reviewed by team members to ensure that it is clear to understand.
2. The work has been updated onto the M2 Report, under Section 3.3 Sequence Diagram.
3. The project timeline remains unaffected, and it continues to progress as per the established schedule.
4. No, there are no changes to the success or completion criteria of the task. The criteria remain as originally defined.
5. Start date 20/10/2023 end date 25/10/2023
